cloudfoundry/repo/repo_manager.go

```diff
@@ -54,57 +51,48 @@ func (rm *Manager) GetGitRepository(repoURL string, user, password, privateKey *
 
 	var r *git.Repository
 
-	urlPath, err := url.Parse(repoURL)
+	p, err := ioutil.TempDir("", "terraform-provider-cf")
```

Comment:
I would rename to `terraform-provider-cf` to `terraform-provider-cloudfoundry` as we are planning to rename the provider.